### PR TITLE
fix(DB/Raid):improved Vault of Archavon

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Raid.

How to test:
Apply SQL
Restart Server and delete your Cache.
Make sure to add basic tanking gear towards Raid ilevel.

Before you start checking damage:
.mod hp 500000
.learn 1908 (Uber heal over time)

If there is any issue with damage and if you get instantly killed, lemme know.

Note: You wont be able to survive some mechanics where second tank is required - Use one more account on those fights.

### CODE REVIEW

**Review by Helias — APPROVED** [review]

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1601827483251961900.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+UPDATE `creature_template` SET `mindmg` = 3337, `maxdmg` = 4676, `DamageModifier` = 1.03 WHERE `entry` = 34015;
+UPDATE `creature_template` SET `mindmg` = 5853, `maxdmg` = 8179, `DamageModifier` = 1.03 WHERE `entry` = 34016;
+UPDATE `creature_template` SET `mindmg` = 3438, `maxdmg` = 4724, `DamageModifier` = 1.03 WHERE `entry` = 33998;
+UPDATE `creature_template` SET `mindmg` = 5327, `maxdmg` = 7426, `DamageModifier` = 1.03 WHERE `entry` = 34200;
+UPDATE `creature_template` SET `mindmg` = 3433, `maxdmg` = 4738, `DamageModifier` = 1.03 WHERE `entry` = 38482;
+UPDATE `creature_template` SET `mindmg` = 5962, `maxdmg` = 8186, `DamageModifier` = 1.03 WHERE `entry` = 38483;
+
+/* BOSS */ 
+UPDATE `creature_template` SET `mindmg` = 18420, `maxdmg` = 25250, `DamageModifier` = 1.01 WHERE `entry` = 35013;
+UPDATE `creature_template` SET `mindmg` = 36840, `maxdmg` = 49310, `DamageModifier` = 1.01 WHERE `entry` = 35306;
```
Comment (iyasenai) [inline data/sql/updates/pending_db_world/rev_1601827483251961900.sql:22]: UPDATE `creature_template` SET `mindmg` = 36840, `maxdmg` = 49310, `DamageModifier` = 1.01 WHERE `entry` = 35360;
```suggestion
UPDATE `creature_template` SET `mindmg` = 36840, `maxdmg` = 49310, `DamageModifier` = 1.01 WHERE `entry` = 35360;
```

**Comment by iyasenai** [comment]
I tested the damage and these were the results.
These were the Buffs and the gear I used.
image

Archavon Warder 25   ID:32368
image
Archavon Warder 10  ID:32353
image
Flame Warder 25  ID: 35359
image
Flame Warder  10  ID: 35143
image
Tempest Warder 25  ID: 34016
image
Tempest Warder 10 ID: 34015
image
Tempest Minion 25  ID: 34200
image
Tempest Minion 10 ID:  33998
image
Frost Warder 25  ID: 38483
image
Frost Warder 10  ID: 38482
image

Bosses.

Koralon the Flame Watcher 10:  ID 35013
image
Emalon the Storm Watcher 25  ID: 33994
image
Emalon the Storm Watcher 10  ID: 33993
image
Archavon the Stone Watcher 25:  ID: 31722
image
Archavon the Stone Watcher 10  ID: 31125
image
Toravon the Ice Watcher 25  ID: 38462
image
Toravon the Ice Watcher 10  ID: 38433
image

**Comment by Si1ker** [comment]
@Knindzagxg  is that ok? seems pretty low
ps: take care of the IDs you almost changed another npc instead of koralon


@iyasenai thanks for testing,

File: data/sql/updates/pending_db_world/rev_1601827483251961900.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1601827483251961900.sql]: Sorted, nice catch mate.
Blame my fast typing again xP

**Comment by Knindzagxg** [comment]
Yeah that's ok mate.
Most damage come from mechanics and this is only Psy basic damage.
During fight most of damage will be on Raid due simple aoe and spreading.

**Comment by Knindzagxg** [comment]
Sorted to be like trough progressing.
Damage is now finally sorted, and sorry cause miss typed ID -> Blame quick typing xP

Yet again thanks for testing @iyasenai - Recount always give perfect informations xP

**Comment by Knindzagxg** [comment]
Will be updated again in next few days.

**Comment by FrancescoBorzi** [comment]
> Will be updated again in next few days.

@Knindzagxg please mark this as WIP and remove TBM if that's the case

**Comment by Knindzagxg** [comment]
Yeah, i always do mate.
Was just hr late cause got stuck in to much logs xP

**Comment by Knindzagxg** [comment]
@iyasenai Toravon ready for testing.
Just make sure to get some Trial Gear (Mixed with Heroic items) like you would use for progression in ICC.

**Comment by iyasenai** [comment]
Apply the changes and these were the results

Toravon 25

image

Toravon 10

image

Icc/Toc gear

image

**Comment by Knindzagxg** [comment]
Perfect xP

Thanks once again mate.
Vault sorted, we have to start on Trial and ICC now xP

**Review by Si1ker — APPROVED** [review]

**Pull request merged**